product-mini/platforms/posix/main.c

```diff
@@ -897,8 +898,15 @@ main(int argc, char *argv[])
             goto fail1;
         }
 
-        bh_memcpy_s(wasm_file_mapped, wasm_file_size, wasm_file_buf,
-                    wasm_file_size);
+#if (WASM_MEM_DUAL_BUS_MIRROR != 0)
+        daddr = os_get_dbus_mirror(wasm_file_mapped);
+#else
+        daddr = wasm_file_mapped;
+#endif
+        bh_memcpy_s(daddr, wasm_file_size, wasm_file_buf, wasm_file_size);
+#if (WASM_MEM_DUAL_BUS_MIRROR != 0)
+        os_dcache_flush();
+#endif
```

Comment:
Maybe we can add an API in core/shared/util (like bh_common.c/h) to copy data into dst addr? So as to simplify the code in aot_loader.c, aot_reloc_xtensa.c and here, which are controlled by macro WASM_MEM_DUAL_BUS_MIRROR.